styles/styles.css

```diff
@@ -6,3 +6,148 @@
  * 
  *
  */
+
+/* main > div {
+  width: 100%!important;
+  margin: 0 auto;
+  max-width: 1260px;
+  padding-left: 70px!important;
+  padding-right: 70px!important;
+} */
+
+body {
+  line-height: 21px;
+}
+
+body table {
+  font-size: 16px;
+  border-spacing: 0;
+}
+
+table, th, td {
+  padding: 9px 10px;
+  border: 1px solid #bdbdbd!important;
+  border-collapse: collapse!important;
+}
+
+table th {
+  background-color: #ededed;
+}
+
+.columns {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.accordion dd div {
+  padding-top: 10px;
+}
+
+.content {
+  display: block;
+  max-width: var(--grid-container-width);
+  margin: 0 auto;
+}
+
+/*
+* title
+*/
+
+.page-title {
+  padding: 10px!important;
+  font-size: 36px;
+  font-weight: 500;
+  color: white!important;
+}
+
+/*
+  internal banner
+*/
+
+main .internal-banner .banner:before {
+  background: url(data:image/svg+xml,%3Csvg%20id%3D%22Layer_1%22%20data-name%3D%22Layer%201%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2018%2018%22%3E%3Cdefs%3E%3Cstyle%3E.cls-1%7Bfill%3A%23fff%3B%7D%3C%2Fstyle%3E%3C%2Fdefs%3E%3Ctitle%3EArtboard%203%3C%2Ftitle%3E%3Cpath%20id%3D%22_Compound_Path_%22%20data-name%3D%22%26lt%3BCompound%20Path%26gt%3B%22%20class%3D%22cls-1%22%20d%3D%22M9%2C2.44a7%2C7%2C0%2C1%2C0%2C7%2C7A7%2C7%2C0%2C0%2C0%2C9%2C2.44ZM8%2C5.94a.5.5%2C0%2C0%2C1%2C.5-.5h1a.5.5%2C0%2C0%2C1%2C.5.5v1a.5.5%2C0%2C0%2C1-.5.5h-1a.5.5%2C0%2C0%2C1-.5-.5Zm2.42%2C7.5H7.63s-.58%2C0-.58-.5.58-.5.58-.5H8v-3H7.6S7%2C9.42%2C7%2C8.94s.58-.5.58-.5H9.8a.2.2%2C0%2C0%2C1%2C.2.2v3.8h.42s.58%2C0%2C.58.5S10.42%2C13.44%2C10.42%2C13.44Z%22%2F%3E%3C%2Fsvg%3E) left no-repeat;
+  background-size: 18px;
+  content: " ";
+  height: 18px;
+  width: 18px;
+  padding: 0 9px;
+  margin-right: 5px;
+}
+
+main .internal-banner .banner {
+  margin: 0 auto;
+  text-align: center;
+  background-color: #0f996d;
+  font-size: 14px;
+  font-weight: 700;
+  padding: 12px 0;
+  line-height: 24px;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+
+main .last-updated.content {
+  font-weight: lighter;
+  font-size: 16px;
+  color: #4b4b4b;
```

Comment:
Could this colour value please be picked from CSS constants instead of hard-coding it here